tox.ini

```diff
@@ -69,6 +68,9 @@ setenv =
     dev: PYTHONPATH = .
     dev: APP_URL = {env:APP_URL:http://localhost:5000}
     dev: WEBSOCKET_URL = {env:WEBSOCKET_URL:ws://localhost:5001/ws}
+    dev: ENABLE_WEB = {env:ENABLE_WEB:true}
+    dev: ENABLE_WEBSOCKET = {env:ENABLE_WEBSOCKET:true}
+    dev: ENABLE_WORKER = {env:ENABLE_WORKER:true}
```

Comment:
This is necessary now that these switches have been added to `supervisord-dev.conf`. Otherwise no processes would run by default when you ran `make dev`